src/contracts/Staking.sol

```diff
@@ -212,37 +219,43 @@ contract Staking is OwnableUpgradeable, StakingStorage {
      */
     function shouldPauseInstantUnstaking(bool _shouldPause) external onlyOwner {
         isInstantUnstakingPaused = _shouldPause;
-        emit LogSetPauseInstantUnstaking(block.number, _shouldPause);
+        emit LogSetPauseInstantUnstaking(_shouldPause);
     }
 
     /**
         @notice set epoch duration
         @dev epoch's determine how long until a rebase can occur
-        @param duration uint
+        @param _duration uint
      */
-    function setEpochDuration(uint256 duration) external onlyOwner {
-        epoch.duration = duration;
-        emit LogSetEpochDuration(block.number, duration);
+    function setEpochDuration(uint256 _duration) external onlyOwner {
+        epoch.duration = _duration;
+        emit LogSetEpochDuration(_duration);
     }
 
     /**
      * @notice set warmup period for new stakers
      * @param _vestingPeriod uint
      */
     function setWarmUpPeriod(uint256 _vestingPeriod) external onlyOwner {
-        require(_vestingPeriod < MAX_VESTING_PERIOD, "Vesting Period too large");
```

Comment:
I did less than to save gas, but it is a call that will barely be called and it saves only 6 gas on and && and == operator, so I think it's fine to change for clarity sake